packages/backend/src/open_payments/quote/errors.ts

```diff
@@ -2,7 +2,8 @@ export enum QuoteError {
   UnknownPaymentPointer = 'UnknownPaymentPointer',
   InvalidAmount = 'InvalidAmount',
   InvalidReceiver = 'InvalidReceiver',
-  InactivePaymentPointer = 'InactivePaymentPointer'
+  InactivePaymentPointer = 'InactivePaymentPointer',
+  FeeExceedsReceiveAmount = 'FeeExceedsReceiveAmount'
```

Comment:
I think it's a problem in the case of fixed send, where this is thrown.

This error is thrown from a check I moved from the MASE's quote endpoint. Thrown when fees are taken out of the receiveAmount for a fixed send (so when the receive amount would be negative): https://github.com/interledger/rafiki/blob/913bc6a91130254fe042242997fd3f3623f87d6d/localenv/mock-account-servicing-entity/app/routes/quotes.ts#L59-L61

I wonder is if this somehow handled elsewhere in the backend though. Or behaves as intended when not handled. Because if not I it means we currently just trust the the ASE to do this check.

Reply:
Oh, I get it. I want to do a fixed send of $1 but the fees would be $1.1.

> Maybe rename to `FeeExceedsFixedSendReceiveAmount`, `NegativeReceiveAmount` or something else?

I like `NegativeReceiveAmount` because its very obvious.

Reply:
updated to `NegativeReceiveAmount`